reference-stores/nuxt/queries/site.js

```diff
@@ -11,33 +11,37 @@ export const SITE_QUERY = `
         }
       }
     }
-    header: content(filter: { type: "componentHeader", handles: ["component-header"] }){
+    header: allContent(filter: { type: "componentHeader", handles: ["component-header"] }){
       ${CONTENT_QUERY_FRAGMENT}
     }
-    newsletter: content(filter: { type: "componentNewsletter", handles: ["component-newsletter"] }){
+    newsletter: allContent(filter: { type: "componentNewsletter", handles: ["component-newsletter"] }){
```

Comment:
For consistency & readability:

```suggestion
    newsletter: allContent(filter: { type: "componentNewsletter", handles: ["component-newsletter"] }) {
```